### ISSUE TITLE
Remote target (TensorFlow Lite & Micro)

### ISSUE THREAD

**Issue by onyx22574** [mlflow/mlflow#4505]
I wonder if there is the possibility to use mlflow for embedded machine learning where ml-models are trained and afterward optimized with certain quantization methods with the use of e.g. TensorFlow-Lite Converter and are deployed to remote target (bare-metal or embedded Linux device). I am looking to track their accuracy vs. the original floating-point model (reference) and its latency. I'd like to monitor this over time and improve it by retraining, acquiring more real-time data, and its use case deployment as well there are multiple remote target scenarios to execute (so-called delegates - TensorFlow terminology) 
I'd like to have it distinguishable (configurable) within or per mlflow project.

Is it possible to use mlflow to deploy, monitor, and track ml-model over full ml life-cycle for remote embedded devices (e.g. RPi-class, bare-metal device, freeRTOS, etc..)?
Is it possible to start off with pre-trained models (e.g. tfhub)?

If so, can you refer to some mlflow examples on embedded machine learning and its deployment for embedded remote target?

Thank you.

**Comment by B-Step62** [comment]
Deploying/tracking on embedded or bare-metal targets (RPi, freeRTOS, TFLite Micro) is outside MLflow 's scope. Closing, feel free to reopen if there is a specific tracking use case.
